includes/class-wc-stripe-apple-pay-registration.php

```diff
@@ -178,43 +224,43 @@ private function register_domain_with_apple( $secret_key = '' ) {
 	 * @return bool True on success, false on failure.
 	 */
 	public function update_domain_association_file( $force = false ) {
-			$path     = untrailingslashit( $_SERVER['DOCUMENT_ROOT'] );
-			$dir      = '.well-known';
-			$file     = 'apple-developer-merchantid-domain-association';
-			$fullpath = $path . '/' . $dir . '/' . $file;
-
-			$existing_contents = @file_get_contents( $fullpath );
-			$new_contents = @file_get_contents( WC_STRIPE_PLUGIN_PATH . '/' . $file );
-			if ( ( ! $existing_contents && ! $force ) || $existing_contents === $new_contents ) {
-				return true;
-			}
+		$path     = untrailingslashit( $_SERVER['DOCUMENT_ROOT'] );
```

Comment:
Is update_domain_association_file called from anywhere? I thought we were going to remove this entirely in favor of exclusively using the rewrite rule

Reply:
Personally, I don't have quite enough information to say confidently that rewrite rules would work in all cases where the file copy does. That said, I'm open to removing it, and will add a commit to do so. In WCPay we'll have an opportunity to measure the failure rate (if any) for the rewrite rule, and can consider (re-)adding a fallback at that point.

Reply:
> I'm open to removing it, and will add a commit to do so

Sounds good

> In WCPay we'll have an opportunity to measure the failure rate (if any) for the rewrite rule

Good idea. Please open an issue if we don't have one already there so we don't forget

Reply:
Removed this in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1357/commits/21cc4b60bf18e1503c731eceda4038c49753251f

Reply:
Left a note about the measurement in 409-gh-Automattic/woocommerce-payments-server#issuecomment-722672016